### PR TITLE
Playtest changes and textbox tweaks

### PR DESCRIPTION
**Pull request by triggrhappy0**

### CODE REVIEW

File: pkto_cockatrice.xml [hunk 1]
```diff
@@ -1366,7 +1366,7 @@ Whenever Exeggutor blocks a Pokémon with flying, put two -1/-1 counters on that
 			<type>Pokémon - Fighting</type>
 			<pt>4/3</pt>
 			<tablerow>2</tablerow>
-			<text>Whenever Hitmonlee attacks, you may have it deal damage equal to its power to target Pokémon. If you do, prevent all combat damage Hitmonlee would deal this turn.</text>
+			<text>Whenever Hitmonlee attacks, you may have it deal damage equal to its power to target Pokémon. If you do, prevent all combat damage Hitmonlee would deal this turn and it doesn't untap during your next untap step.</text>
```
Comment (brooks42) [inline pkto_cockatrice.xml:1369]: No change to the card art though?

File: pkto_cockatrice.xml [hunk 2]
```diff
@@ -1904,7 +1905,7 @@ When you cycle Mew, shuffle it into your library.</text>
 			<type>Legendary Trainer - Blue</type>
 			<loyalty>4</loyalty>
 			<text>At the beginning of your upkeep, if a player attacked you last turn, draw a card.
-[+1]: The next Pokémon spell you cast this turn costs {1} less to cast for each Pokémon your opponents control.
+[+1]: If a player controls more Pokémon than you, the next Pokémon spell you cast this turn costs {X} less to cast, where X is equal to the difference.
```
Comment (brooks42) [inline pkto_cockatrice.xml:1908]: Interesting wording, nice

File: pkto_cockatrice.xml [hunk 3]
```diff
@@ -2369,7 +2370,7 @@ When there are four or more charge counters on Dowsing Machine, sacrifice it.</t
 			<name>Disable</name>
 			<set picURL="https://brooks42.github.io/pkto/cards/Disable.png" rarity="common">PKTO</set>
 			<color>UB</color>
-			<manacost>{U/B}</manacost>
+			<manacost>U{U/B}</manacost>
```
Comment (brooks42) [inline pkto_cockatrice.xml:2373]: UU/B instead of 1U/B huh 🤔

File: pkto_cockatrice.xml (hunk 2 above)
Comment (triggrhappy0) [reply pkto_cockatrice.xml:1908]: Oops, added

File: pkto_cockatrice.xml (hunk 3 above)
Comment (triggrhappy0) [reply pkto_cockatrice.xml:2373]: Yeah I think counterspells should require blue.

Comment (brooks42) [reply pkto_cockatrice.xml:2373]: https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=126812

;P

Comment (brooks42) [reply pkto_cockatrice.xml:2373]: I think Disable should draw a card if it requires blue. Or at least rummage?

Comment (brooks42) [reply pkto_cockatrice.xml:2373]: image

image

I think rummaging would be ideal, but it's a lot of text :/

Comment (triggrhappy0) [reply pkto_cockatrice.xml:2373]: Stop citing planar chaos lol, it doesn't count. I think rummaging would be fine to add, but it is a lot of text. Even if it wasn't itty bitty, it's just going to be a lot to process on one card. I think countering a thing for two turns is pretty good already.

**Pull request merged**